lib/java/logging-service-telemetry/src/main/java/org/enso/logging/service/telemetry/LogJobsProcessor.java

```diff
@@ -123,6 +124,7 @@ private void sendBatch(List<LogJob> batch) throws RequestFailureException {
       }
       if (refreshedAuthData != null) {
         authenticationData = refreshedAuthData;
+        LOGGER.trace("Token refreshed successfully: {}", authenticationData);
```

Comment:
The `authenticationData` will contain the token. 
I don't think it's good practice to include sensitive information like that in the logs.

I think it would be enough to just log that it was successful and perhaps include the expiration date.

Reply:
1) It is in TRACE level, which is not included anywhere, by default. It is only useful once you explicitly increase the verbosity of the logging framework.
2) Credentials are already stored in `$HOME/.enso/credentials` by the IDE. I don't think there is any sensitivity violation.

Reply:
> 2. Credentials are already stored in `$HOME/.enso/credentials` by the IDE. I don't think there is any sensitivity violation.

The `credentials` file is even by name expected to contain such information, people won't share it easily. But people may share logs when troubleshooting issues, even on trace levels - they may need to enable _trace_ for some other things to appear and may not expect that they accidentally expose sensitive information that way.

Actually we had a whole 'log masking' (`org.enso.logger.masking`) implementation to avoid accidentally leaking sensitive information. Maybe you could try using that to hide the access token then?

Reply:
Fair point. Logging only new expiration in 6e1ba187bf6e72059a6dad7aae039efff227a778